com.unity.netcode.adapter.utp/Runtime/UnityTransport.cs

```diff
@@ -161,6 +161,7 @@ public struct ConnectionAddressData
         private NetworkPipeline m_UnreliableFragmentedPipeline;
         private NetworkPipeline m_UnreliableSequencedFragmentedPipeline;
         private NetworkPipeline m_ReliableSequencedFragmentedPipeline;
+        private const byte k_PipelineCount = 3; //Represented the number of existing pipeline
```

Comment:
Better than this constant would be to have an enum something like the following:
```
enum Pipelines
{
  UnreliableSequence,
  ReliableFragmented,
  NonSequenced,
  Count
}
```